packages/mds-transactions-service/service/validators.ts

```diff
@@ -1,10 +1,43 @@
 import Joi from 'joi'
 import { schemaValidator } from '@mds-core/mds-schema-validators'
-import { TransactionDomainModel } from '../@types'
+import { TransactionDomainModel, TransactionOperationDomainModel, TransactionStatusDomainModel } from '../@types'
 
-export const { validate: validateTransactionDomainModel, isValid: isValidTransactionDomainModel } = schemaValidator<TransactionDomainModel>(
+export const { validate: validateTransactionDomainModel, isValid: isValidTransactionDomainModel } = schemaValidator<
+  TransactionDomainModel
+>(
   Joi.object<TransactionDomainModel>().keys({
-    name: Joi.string().required(),
-    text: Joi.string().required()
+    transaction_id: Joi.string().uuid().required(),
+    provider_id: Joi.string().uuid().required(),
+    device_id: Joi.string().uuid().optional(),
```

Comment:
`.optional()` is the default if you don't specify `.required()` ... it can be omitted.

Reply:
hm, i like it explicit.